### ISSUE TITLE
Can't find name of "Class"

### ISSUE THREAD

**Issue by odex21** [johnsoncodehk/volar#274]
I used the library Revogrid
Its plugin requires passing Class as a parameter

![图片](https://user-images.githubusercontent.com/38051167/122681531-4948d700-d227-11eb-865d-eeeff613aabd.png)
![图片](https://user-images.githubusercontent.com/38051167/122681549-58c82000-d227-11eb-8b57-01255ee398c8.png)

**Comment by johnsoncodehk** [comment]
How are you expose `StrethchPlus` to template?

**Comment by odex21** [comment]
> 
> 
> How are you expose `StrethchPlus` to template?

just in `<script setup lang="ts">`

**Comment by odex21** [comment]
Well, I found out that the Class that comes in from import is fine. But I need this Class to get the Context, the `columns` from the props, not the already processed ones from the component.

I know it's a bit weird to bind with `Class` and `Context`, which is not a good practice to say the least, but for now it's easier to implement this way. So I also raised a related issue to reveogrid. https://github.com/revolist/revogrid/issues/220.
